code/obj/mining.dm

```diff
@@ -897,6 +897,7 @@ TYPEINFO_NEW(/turf/simulated/wall/auto/asteroid)
 	var/mining_toughness = 1 //Incoming damage divided by this unless tool has power enough to overcome.
 	var/topnumber = 1
 	var/orenumber = 1
+	var/list/topoverlaycache = list()
```

Comment:
Maybe better in New() or somewhere else, thinking about hidden init proc?

Reply:
Ight